templates/course-results/lessons.php

```diff
@@ -185,9 +189,11 @@
 	<?php
 	/**
 	 * Fires inside course-results/lessons.php after the
-	 * is uer logged check, at the bottom of all lessons.
+	 * is user logged check, at the bottom of all lessons.
 	 *
 	 * @since 1.4.0
+	 *
+	 * @hook sensei_course_results_after_lessons
```

Comment:
```suggestion
	 * @hook sensei_course_results_after_lessons
	 *
	 * @param {int} $course_id The course ID.
```

It looks like the `$course_id` param is missing.